File: flutter_exam/lib/model/place.dart
class Place {
  final int id;
  final String name;
  final String picture;
  final int rating;

  Place({required this.id, required this.name, required this.rating, required this.picture});

  factory Place.fromJson(Map<String, dynamic> json) {
    return Place(
      id: json['id'],
      name: json['name'],
      rating: json['rating'],
      picture: json['picture'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'rating': rating,
      'picture': picture,
    };
  }
}

File: flutter_exam_api/src/main/java/com/project/flutter_exam/model/Place.java
package com.project.flutter_exam.model;

public class Place {
    private int id;
    private String name;
    private String picture;
    private int rating;

    public Place(int id, String name, String picture, int rating) {
        this.id = id;
        this.name = name;
        this.picture = picture;
        this.rating = rating;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPicture() {
        return picture;
    }

    public void setPicture(String picture) {
        this.picture = picture;
    }

    public int getRating() {
        return rating;
    }

    public void setRating(int rating) {
        this.rating = rating;
    }
}
